src/main/java/uk/gov/ccs/swagger/sso/api/OrganisationUserApi.java

```diff
@@ -65,11 +65,11 @@ public void setApiClient(ApiClient apiClient) {
      * @return Call to execute
      * @throws ApiException If fail to serialize the request body object
      */
-    public com.squareup.okhttp.Call organisationsOrganisationIdUsersGetCall(String organisationId, Integer pageSize, Integer currentPage, String searchString, Boolean includeSelf, final ProgressResponseBody.ProgressListener progressListener, final ProgressRequestBody.ProgressRequestListener progressRequestListener) throws ApiException {
+    public com.squareup.okhttp.Call organisationProfileorganisationIdUsersGetCall(String organisationId, Integer pageSize, Integer currentPage, String searchString, Boolean includeSelf, final ProgressResponseBody.ProgressListener progressListener, final ProgressRequestBody.ProgressRequestListener progressRequestListener) throws ApiException {
         Object localVarPostBody = null;
         
         // create path and map variables
-        String localVarPath = "/organisations/{organisationId}/users"
+        String localVarPath = "/organisation-profile{organisationId}/users"
```

Comment:
@jalvares-crown @tberey - is there a missing '/' between organisation-profile and {organisationId}?

Reply:
@ccs-wattsa It looks like you have reviewed some old outdated files which don't exist in the latest version of the github repo. A good way to confirm this is to checkout the code and look for the line of code you have commented on. The Github interface can be misleading sometimes.